lib/command.ts

```diff
@@ -233,15 +236,20 @@ export function command(
       const builderOutput = builder(yargs.reset(parsed.aliases));
       const innerYargs = isYargsInstance(builderOutput) ? builderOutput : yargs;
       if (shouldUpdateUsage(innerYargs)) {
-        innerYargs
-          .getUsageInstance()
-          .usage(
-            usageFromParentCommandsCommandHandler(
-              parentCommands,
-              commandHandler
-            ),
-            commandHandler.description
-          );
+        if (!command) {
+          tmpCommands.forEach(e => innerYargs.getUsageInstance().command(...e));
+          tmpUsage.forEach(e => innerYargs.getUsageInstance().usage(...e));
+        } else {
```

Comment:
This fix seems reasonable to me, we will need to rebase with #1823; which is close to landing and touches this same code.

Reply:
Merged with master.